textproc/dwdiff/Portfile

```diff
@@ -23,16 +23,25 @@ long_description    \
 homepage            https://os.ghalkes.nl/dwdiff.html
 master_sites        https://os.ghalkes.nl/dist/
 
-checksums           rmd160  e0a7eedabbc500731bd85b8c77c9716c18889818 \
-                    sha256  c1c9217825ad3fac9936129cc26b73fa5479fb079e5c34522fc7066e74e31ef8
+checksums           rmd160  a856e73129ca652197daf61bf92c1491ca4d8328 \
+                    sha256  df16fec44dcb467d65a4246a43628f93741996c1773e930b90c6dde22dd58e0a \
+                    size    79857
 
-extract.suffix      .tgz
+extract.suffix      .tar.bz2
```

Comment:
You also need to add `use_bzip2 yes`, to ensure `lbzip2` is installed if necessary.

Reply:
> You also need to add `use_bzip2 yes`, to ensure `lbzip2` is installed if necessary.

Otherwise, looks good.